scripts/hid_configurator/modules/dfu.py

```diff
@@ -209,7 +209,13 @@ def b0_get_dfu_image_bootloader_var():
 
 
 def mcuboot_common_is_dfu_file_correct(dfu_bin):
-    res, _, _ = imgtool.image.Image.verify(dfu_bin, None)
+    try:
+        res, _, _ = imgtool.image.Image.verify(dfu_bin, None)
```

Comment:
should we still support the outdated `verify` API?

Reply:
It's not "outdated" but official". As a current workaround, we catch exception and retry with 4 returned values, because  imgtool version from `sdk-mcuboot` broke backwards compatibility and returns 4 values instead of 3

Reply:
The official is the one that we refer to in the installation guide. The PyPI version is irrelevant, as you always get the correct version if you set up the NCS correctly or according the installation instructions

Reply:
"official" is the one from PyPI (https://pypi.org/project/imgtool/)
Temporarily, we need to use imgtool from source (https://github.com/nrfconnect/sdk-mcuboot/tree/main/scripts) to support pure ED25519 (needed for our nRF54L use-case). Unless I missed something, setting up NCS would not result in installing the "from source" version automatically.

Reply:
discussed offline, irrelevant